project/plugins.sbt

```diff
@@ -1,4 +1,4 @@
-addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.9")
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.13")
```

Comment:
This should be kept at 0.6.9 so that people using older Scala.js versions can use the library.